README.md

```diff
@@ -1,4 +1,4 @@
-# Leaderboard: hit the API
+# Leaderboard: final touches
```

Comment:
- [ ] Great job. kindly add the steps and procedures needed to run your app in the readme file. great job